### PR TITLE
Getting Started and What's New dashboards are now removed.

### LINKED ISSUE #5049: Unused code and assets from the old Getting Started screen should be removed
- [x] Getting Started tab and assets should be removed
- [x] What's New tab and assets should be removed

### PR DESCRIPTION
**Pull request by kjohnson**
Resolves #5049 

## Description

- Deletes `admin/welcome.scss`.
- Removes `@import 'welcome'` from `admin/give-admin.scss`.
- Deletes `admin/class-give-welcome.php`.
- Removes `assertFileExists` for 'class-give-welcome.php`.
- Removes `admin/class-give-welcomes.php` from `admin/class-give-admin.php`.
- Removes the `welcome`/"What's New" setting from `class-settings/advanced.php`.
- Removes `assertArrayHasKey` for `welcome`.
- Removes `disable_welcome` from 'upgrade-functions.php`.
- Removes `welcome` from 'install.php`.
- Removes references in `give_admin_rate_us()`.
- Removes references in `give_stripe_show_connect_banner()`.

## Affects

This may need to remove the `disable_welcome` key from `give_v18_renamed_core_settings()` in `upgrade-functions.php` - I'm not sure if this is a backwards compatibility issue.

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [ ] Tests included
-   [ ] Keyboard accessible
-   [ ] Screen reader accessible
-   [ ] Relevant `@since` tags included in DocBlocks
-   [ ] Changelog updated
-   [ ] Labels applied if docs are needed
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: includes/admin/upgrades/upgrade-functions.php [hunk 1]
```diff
@@ -1043,7 +1043,7 @@ function give_v18_renamed_core_settings() {
 	return [
 		'disable_paypal_verification' => 'paypal_verification',
 		'disable_css'                 => 'css',
-		'disable_welcome'             => 'welcome',
+		'disable_welcome'             => 'welcome', // @TODO: Not sure if this needs to be removed, or kept for backwards compatibility.
```
Comment (henryholtgeerts) [inline includes/admin/upgrades/upgrade-functions.php:1046]: @ravinderk Do you know of any addons that rely on/check for this setting?

**Comment by kjohnson** [comment]
Update:

- Removes references in `give_admin_rate_us()`.
- Removes references in `give_stripe_show_connect_banner()`.

File: includes/admin/upgrades/upgrade-functions.php (hunk 1 above)
Comment (ravinderk) [reply includes/admin/upgrades/upgrade-functions.php:1046]: @kjohnson It is safe to remove.

Comment (kjohnson) [reply includes/admin/upgrades/upgrade-functions.php]: Removed.

**Comment by kjohnson** [comment]
Removed welcome page `gif`s: 050a78b64

**Pull request merged**